src/Game/Renderer.java

```diff
@@ -1015,7 +1017,8 @@ public static void present(Graphics g, Image image) {
         // TODO: This will need to be adjusted when the login screen is resizable
         Rectangle bounds;
         if (Client.login_screen == Client.SCREEN_USERNAME_PASSWORD_LOGIN) {
-          bounds = new Rectangle(512 - 148, 346 - 36, 48, 16);
+        	bounds = new Rectangle(512 - 100, 216, 48, 16);
```

Comment:
this is a silly location for the -server replay- buttons & it doesn't work because some status messages overflow underneath the buttons. The buttons will have to be rethought before a new release is made (but can be rethought after merge)

Reply:
I'm working on this, you don't need to touch it.

Reply:
alright, perfect